src/Fixer/Strict/StrictTypesFixer.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Strict;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ * @author SpacePossum
+ */
+final class StrictTypesFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return count($tokens) > 1 && $tokens[0]->isGivenKind(T_OPEN_TAG);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        // check for if declaration already in file
+        $sequence = $this->getDeclareStrictTypeSequence();
+        $searchIndex = $tokens->getNextMeaningfulToken(0);
+        if (null === $searchIndex) {
+            $this->insertNewSequence($tokens); // not found, insert declare statement
```

Comment:
`$searchIndex` is `null`, so there is no meaningful token in file (which should not be happen because of `isCandidate` check). But then you add the declare sequence (without php open tag).
I think something is wrong here..